server/sandbox_list.go

```diff
@@ -60,7 +60,7 @@ func (s *Server) ListPodSandbox(ctx context.Context, req *pb.ListPodSandboxReque
 		}
 
 		rStatus := pb.PodSandboxState_SANDBOX_NOTREADY
-		if sb.Ready(false) {
+		if sb.Ready(true) {
```

Comment:
Concern with this is that PLEG may go unhealthy if some pod operation takes long.

Reply:
Is it due to runtimeOCI#UpdateContainerStatus taking long ?
See update-cont-stat.txt below for proposal on reducing the duration opLock is held.

Reply:
We skip over sandboxes that aren't created a few lines above. So, not-ready, not-created sandboxes shouldn't show up here.

Reply:
See #3650

Reply:
Is it possible that the sandbox has exited by the time we retrieve the state ?

Reply:
I feel like this change is out of scope for the PR. Is there a reason to have it here?

Reply:
@haircommander 
I have taken this out of the PR.